common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -2686,6 +2686,8 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
         "Estimate statistics in absence of statistics."),
     HIVE_STATS_NDV_ESTIMATE_PERC("hive.stats.ndv.estimate.percent", (float)20,
         "This many percentage of rows will be estimated as count distinct in absence of statistics."),
+    HIVE_STATS_JOIN_NDV_READJUSTMENT("hive.stats.join.ndv.readjustment", false,
+        "Setting this to true will make Hive use Calcite to adjust estimatation for ndv after join."),
```

Comment:
`Setting this to true will make Hive use Calcite`

Instead of 'Calcite' logic, could you maybe mention (one-liner since it is a config and there is no need of it to be too long) the kind of estimation it does?